tools/update-tools/update_tools.py

```diff
@@ -929,7 +930,9 @@ def __init__(self, fstab, sdk):
             self.import_releasetools()
             if self.sdk_version >= 21:
                 self.itemset = ItemSet("system", "META/filesystem_config.txt")
-        self.generator = edify_generator.EdifyGenerator(1, {"fstab": self.fstab})
+        if bool(os.environ.get('USE_SET_METADATA', False)) == True:
+            self.info_dict['use_set_metadata'] = True
```

Comment:
nit: Let's add a comment that this enables the set_metadata() and set_metadata_recursive() functions in the generated script